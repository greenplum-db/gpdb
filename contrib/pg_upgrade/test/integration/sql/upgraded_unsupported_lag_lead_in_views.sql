SELECT 1 FROM lagview1;
SELECT 1 FROM leadview1;
