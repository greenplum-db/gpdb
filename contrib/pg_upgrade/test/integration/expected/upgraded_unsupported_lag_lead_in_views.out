SELECT 1 FROM lagview1;
 ?column? 
----------
 1        
(1 row)
SELECT 1 FROM leadview1;
 ?column? 
----------
 1        
(1 row)
